src/light_pcapng_ext.c

```diff
@@ -517,6 +525,11 @@ int light_write_packet(light_pcapng pcapng, const light_packet_interface* packet
 		light_option dropcount_opt = light_create_option(LIGHT_OPTION_EPB_DROPCOUNT, 8, &packet_header->dropcount);
 		light_add_option(NULL, packet_block_pcapng, dropcount_opt, false);
 	}
+	if (packet_header->queue)

```

Comment:
0 is a valid queue value or not?

Reply:
I think it's the default value. So no queue means it's 0